### DISCUSSION TITLE
Outline looks great, but the lack of email authentication makes it a no for me

### DISCUSSION THREAD

**Discussion by tonycoculuzzi** [outline/outline#3232] [General] [answered]
Is email authentication ever going to be in the plans? Outline looks close to what we want, but email authentication is a must have for us and without it we can't use Outline as an organization.

**Comment by tommoor** [comment, marked as answer]
To be clear, once a team is created you can invite users to signin with email + magic link. It is not a requirement that everyone uses SSO.

**Reply by almereyda** [reply]
Thank you for pointing this out! I have tested this on our instance running v0.62.0, which contains #2985, and have been able to login as an invited user that is not part of the OIDC IP and has another email domain than the Team. The instance is configured without setting `ALLOWED_DOMAINS`.

**Reply by tonycoculuzzi** [reply]
Ah I didn't realize that, that's good to hear!

Now if I could only get it working! 😅
